### ISSUE TITLE
File path returns {}, {}, {}, {}

### ISSUE THREAD

**Issue by prkindia** [follow2vivek/StoragePath#17]
Code
`files = await StoragePath.videoPath;`
The output
"files":[{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{},{}....

**Comment by faisalansari0367** [comment]
yes it works fine with profile and debug mode but in release mode it returns empty

**Comment by prkindia** [comment]
That's the real issue @faisalansari0367

**Comment by kevin4dhd** [comment]
any solution?
